CHANGELOG.md

```diff
@@ -2732,4 +2732,16 @@
   - To differentiate between 'runbook' actions, and 'VM-specific' actions (i.e., those dictated in the ```VMs``` datastructure), tasks that are VM specific have been updated to indicate 'VM-specific' in their name.
   - Added ```Before``` and ```After``` sections to the ```VMs.OS.PostInstall.Commands``` object.  The ```Before``` commands are executed before any VM-specific action is performed (i.e. Create groups, install packages, render templates), while the ```After``` commands are executed at the very end of the VM build process, after all other VM-specific actions are performed.
   - Removed unnecessary command from ```templates/Ubuntu_Server_24.04_runbook.j2```.
-  - Renamed ```FinalDestination``` to ```Destination``` in the ```VMs``` data structure.
\ No newline at end of file
+  - Renamed ```FinalDestination``` to ```Destination``` in the ```VMs``` data structure.
+  
+## Dev-v8.0.0 10-OCTOBER-2024
+
+### Added by Aaron Ellis
+  - Added support for deploying an SDDC Manager appliance during POD Build. Can be used for VCF Import, possibly automated later. 
+	- Added SDDC Manager v5.2 to the ```software_sample.yml``` file
+	- Added Nested_SDDC section to the ```config_sample.yml``` file
+	- Updated ```templates/Pod_Config.j2```
+	- Updated ```CreatePodConfig.yml```
+	- Added ```playbooks/DeploySDDCMgr.yml```
```

Comment:
A small detail but I agree with Luis feedback. Let's go for playbooks/DeploySDDCManager.yml for improved readability.